landing_page/index.html

```diff
@@ -73,7 +73,10 @@
                     aria-controls="navbar">
                     <i></i>
                 </a>
-                <a class="navbar-brand" href="index.html">AdaLite Wallet</a>
+                <div style="display: flex; flex-direction: column;">
+                  <a class="navbar-brand" style="padding-bottom: 2px;" href="index.html">AdaLite Wallet</a>
+                  <span class="navbar-brand" style="font-size: 18px; cursor: default; padding: 0; margin: 0;height: 10px; bottom: 14px; position: relative;">1.0.0</span>

```

Comment:
As I wrote above, I would remove the version from there. It's unnecessary and easy to forget to maintain.